localtime/data_source_timestamp_test.go

```diff
@@ -0,0 +1,68 @@
+package localtime
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+type TestAccDataSourceTimestampCase struct {
+	Name              string
+	Config            string
+	ExpectedTimestamp string
+}
+
+func TestAccDataSourceTimestamp_basic(t *testing.T) {
+	testAccDataSourceTimestampHelper(t, &TestAccDataSourceTimestampCase{
+		Name:              "basic",
+		ExpectedTimestamp: "2020/11/22 11:11:11 -0800",
+		Config: `
+		data "localtime_timestamp" "basic" {
+		  local_time = "2020/11/22 11:11:11"
+		  location   = "America/Los_Angeles"
+		}`,
+	})
+}
+
+func TestAccDataSourceTimestamp_layout(t *testing.T) {
+	testAccDataSourceTimestampHelper(t, &TestAccDataSourceTimestampCase{
+		Name:              "layout",
+		ExpectedTimestamp: "2020-11-22T11:11:11 PST",
+		Config: `
+		data "localtime_timestamp" "layout" {
+		  local_time = "2020/11/22 11:11:11"
+		  location   = "America/Los_Angeles"
+		  layout     = "2006-01-02T15:04:05 MST"
+		}`,
+	})
+}
+
+func TestAccDataSourceTimestamp_timezone(t *testing.T) {
```

Comment:
Would like to see a test case with a date during daylight savings so we can see that `PDT` or `-0700` is rendered for the same location. If I recall correctly, that's the key functionality we're getting out of the provider.